sagemaker-remote-function/pytorch_mnist_sample_notebook/pytorch_mnist.ipynb

```diff
@@ -0,0 +1,558 @@
+{
```

Comment:
<p>Probably best to remove full stop from title?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3926/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>